tests/unit/test_aggregate.py

```diff
@@ -0,0 +1,45 @@
+import os
+import sys
+
+import subprocess as sp
+from tempfile import TemporaryDirectory
+import shutil
+from pathlib import Path, PurePosixPath
+
+sys.path.insert(0, os.path.dirname(__file__))
+
+import common
+
+
+def test_aggregate():
+
+    with TemporaryDirectory() as tmpdir:
+        workdir = Path(tmpdir) / "workdir"
+        data_path = PurePosixPath(".tests/unit/aggregate/data")
+        expected_path = PurePosixPath(".tests/unit/aggregate/expected")
+
+        # Copy data to the temporary workdir.
+        shutil.copytree(data_path, workdir)
+
+        # dbg
```

Comment:
remove before merging. No debugs in tests please!

Reply:
Removed tests: [Reasoning](https://github.com/WayScience/CytoPipe/pull/1#discussion_r846309821)